packages/frontend/core/src/components/affine/setting-modal/workspace-setting/new-workspace-setting-detail/delete-leave-workspace/index.tsx

```diff
@@ -73,18 +82,24 @@ export const DeleteLeaveWorkspace = () => {
       }
     }
 
-    await workspacesService.deleteWorkspace(workspace.meta);
+    if (isOwner) {
+      await workspacesService.deleteWorkspace(workspace.meta);
+    } else {
```

Comment:
Is this behavior confirmed by product team?

Reply:
This is a bug fix, the previous behavior was like this, if it is the owner, delete it, if not, leave the workspace.